android/java/res/values-night/brave_colors.xml

```diff
@@ -276,5 +276,5 @@
     <color name="schemes_surface_container_highest">#303034</color>
     <color name="text_interactive">#7c91ff</color>
 
-    <color name="quick_search_bg">#000000</color>
```

Comment:
Could you please add a comment why this line is commented out instead of being removed? Is it expected it will be required in the future?